whitepaper/chapter01.md

```diff
@@ -44,13 +44,15 @@ There are some additional key principles to cloud native that are important to g
+Cloud native systems have a clear separation between the microservices. The separation is based on a technology like containers which allows independent life cycle management of each microservice. Cloud native applications, which are built on these microservices, and systems generally benefit from design considerations that segregate services where appropriate by leveraging technology such as containers. Additional benefits of cloud native architectures include scalability (vertical and horizontal) per microservice. This simplifies deployment in a wide variety of infrastructure environments, ranging from endpoint device to edge station to large data centres. It also allows service providers to deploy resources in the most efficient manner possible while maintaining necessary service levels for regulated and unregulated service types.
 
 The delivery of software as microservices harnesses the existing group boundaries within an organisation and works with, not against, the different rates of change and business capabilities residing within those boundaries. This enables effective parallel development and communication across an organisation, which allows events such as security patches to be rolled out faster and easier because they don't have to be deployed in lock-step with other parts of the application.
 
-While one microservice per container is the ideal state for cloud native applications, the reality is not always so clean and simple. It is entirely possible and sometimes required to run multiple microservices in a single container, and even to deploy very large containers for aggregations of services that are difficult or impossible to decouple due to software or network limitations. However, the general idea behind cloud native is to decompose microservices into the smallest possible infrastructure unit allowable to run the microservice and shift the burden of orchestration to the orchestration layer.
+While one microservice per container is the ideal state for cloud native applications, the reality is not always so clean and simple. It is entirely possible and sometimes required to run multiple microservices in a single container, and even to deploy very large containers for aggregations of services that are difficult or impossible to decouple due to software or network limitations. However, the general idea behind cloud native is to decompose microservices into the smallest possible infrastructure unit allowable. This causes a shift in where the burden of orchestration lies, and pushes it solely (ideally) into the orchestration layer.
```

Comment:
I like the expansion on the second half with the orchestration layer.

> However, the general idea behind cloud native is to decompose microservices into the smallest possible infrastructure unit allowable.
For the decomposing sentence what drives "what is _allowable_?"   It's lost the reason when splitting.  Something needs to be added to that sentence, IMO.

Reply:
I cleaned up the earlier comment on appropriate segregation, but I need some recommendations on how to provide context to this specific concept. This is a holdover from the original draft, and I'm not 100% sure how to scope the concept of an "allowable unit".

Reply:
Linking to "even to deploy very large containers for aggregations of services that are difficult or impossible to decouple due to software or network limitations" - is the "allowable unit" linked to those "software or network limitations"? And if so, what do we think those limitations are?